test/integration/narrow/example-grant/__snapshots__/country.spec.js.snap

```diff
@@ -228,8 +246,8 @@ exports[`Country Page snapshot should match snapshot 1`] = `
     </div>
     
       
-  <div class="govuk-grid-column-one-third  govuk-!-padding-top-4 govuk-body">
-    <hr class="sideBarColor"/>
+  <div class="govuk-grid-column-one-third  govuk-!-padding-top-0 govuk-body">
+    <hr style="background-color:#1d70b8; height:2px; border:none;">
```

Comment:
Does it not affect the prototype defined styles of the country page?

Reply:
Looks like it affected the styles of earlier country page story as per below:
![Screenshot 2024-11-18 at 09 23 47](https://github.com/user-attachments/assets/0a7ea0b2-6222-44db-9741-0801c3b3194c)